### PR TITLE
{tools}[GCCcore/11.2.0] Clang-AOMP v4.5.0, rocm-cmake v4.5.0, rocminfo v4.5.0, ROCR-Runtime v4.5.0, ...

### PR DESCRIPTION
**Pull request by nordmoen**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by nordmoen** [comment]
This requires the Clang-AOMP easyblock: easybuilders/easybuild-easyblocks#2617

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/ROCT-Thunk-Interface/ROCT-Thunk-Interface-4.5.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+dependencies = [
+    ('libdrm', '2.4.107'),
+    ('numactl', '2.0.14'),
+]
+builddependencies = [
+    ('CMake', '3.21.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+group = 'video'
```
[the remark this answers is not in the document]
Comment (nordmoen) [reply easybuild/easyconfigs/r/ROCT-Thunk-Interface/ROCT-Thunk-Interface-4.5.0-GCCcore-11.2.0.eb:25]: Not sure at all, just based on the requirements that AMD put in ROCT

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2617
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
pg-lab02 - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7542 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/01bae407ce8f14aabff05b9d440b5284 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2617
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
pg-lab02 - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7542 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e43ab497e94214ebb6e67a460a5a8e97 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2617
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3517.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a9ddef80263dbfe95ada21b8acec806e for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @nordmoen!

**Pull request merged**